bokehjs/src/coffee/models/markers/asterisk.coffee

```diff
@@ -1,8 +1,12 @@
 _ = require "underscore"
 Marker = require "./marker"
+bokehgl = require "../glyphs/bokehgl"
 
 class AsteriskView extends Marker.View
 
+  _init_gl: (gl) ->
+    @glglyph = new bokehgl.AsteriskGLGlyph(gl, this)
+
```

Comment:
If `bokehglk.coffee` just exported the standard type names, without the "GL" tacked in the middle, then I think this repeated boilerplate could be reduced to a single method on `Glyph` base class that just does something like:
```
_init_gl: (gl) ->
  @glglyph = new bokehgl[@type](gl, @)
```
would need to be a little more sophisticated, check / log that GL glyph actually exists, but the point is that that would happin in one place.